pkg/kubecost/allocation.go

```diff
@@ -1668,11 +1668,30 @@ func (as *AllocationSet) Clone() *AllocationSet {
 		idleKeys[k] = v
 	}
 
+	var errors []string
+	var warnings []string
+
+	if as.Errors != nil {
+		errors = make([]string, len(as.Errors))
+		copy(errors, as.Errors)
+	} else {
+		errors = nil
+	}
```

Comment:
Just for the future, `nil` is the default value of a slice like `[]string`, so when you declare:
```go
var errors []string
```

`errors` will be `nil` until you change it!